src/gl.rs

```diff
@@ -1112,6 +1112,24 @@ pub fn get_program_iv(program: GLuint, pname: GLenum) -> GLint {
 }
 
 #[inline]
+pub fn get_vertex_attrib_iv(index: GLuint, pname: GLenum) -> GLint {
+    unsafe {
+        let mut result: GLint = 0 as GLint;
+        ffi::GetVertexAttribiv(index, pname, &mut result);
+        return result;
+    }
+}
+
+#[inline]
+pub fn get_vertex_attrib_fv(index: GLuint, pname: GLenum) -> Vec<GLfloat> {
+    unsafe {
+        let mut result = vec![0 as GLfloat; 4];
```

Comment:
The calls seem correct, but we seem to need to raise an error on `getVertexAttrib(0, CURRENT_VERTEX_ATTRIB)`, from the OpenGL man page:

```
Generic vertex attribute 0 is unique in that it has no current state, so an error will be generated if index is 0.
```